### PR TITLE
Member cannot submit events in less than 48 hours from now

### PR DESCRIPTION
**Pull request by Cealtea**
no limit for admin

### CODE REVIEW

File: static/js/edit.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
-$(function() {
-  $(".datepicker").datepicker();
-  $('.datepicker, #roomlist input, #start_time_hour, #start_time_minute, #start_time_ampm, #end_time_hour, #end_time_minute, #end_time_ampm').blur(checkDup);
+$(function () {
+    $(".datepicker").datepicker({
+        minDate: 2
+    });
+    // $('.datepicker, #roomlist input, #start_time_hour, #start_time_minute, #start_time_ampm, #end_time_hour, #end_time_minute, #end_time_ampm').blur(checkDup);
```
Comment (cherukumilli) [inline static/js/edit.js]: do we really need these commented out code?

**Comment by cherukumilli** [comment]
LGTM

File: static/js/edit.js (hunk 1 above)
Comment (Cealtea) [inline static/js/edit.js]: I am not sure why it is commented. Let me check. Good catch!

Comment (Cealtea) [inline static/js/edit.js]: I removed this part and the method below because it is not used

**Pull request merged**